qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/connector/PlacementConnectorImpl.kt

```diff
@@ -43,24 +49,34 @@ internal class PlacementConnectorImpl : PlacementConnector {
     private val LOGGER = QBLogger.getFor("PlacementConnector")
   }
 
-  private val placementAPI = Retrofit.Builder()
-      .baseUrl(BASE_URL_PLACEHOLDER)  // https://stackoverflow.com/questions/34842390/how-to-setup-retrofit-with-no-baseurl
-      .addConverterFactory(GsonConverterFactory.create())
-      .build()
-      .create(PlacementAPI::class.java)
+  private val placementAPI: PlacementAPI
+
+  init {
+    val timeout = configurationRepository.load()?.placementRequestTimeout?.toLong()
```

Comment:
Is it acceptable simplification, that timeout is set at startup only? Changes of configuration in backend doesn't change this setting till next application start.
It is highly probable, that on first startup of application, when request for configuration is not accomplished yet and local configuration cache is empty, default values will be used.

Reply:
fixed